packngo.go

```diff
@@ -135,24 +144,52 @@ type OptionsGetter interface {
 	GetOptions() *GetOptions
 }
 
-func makeSureGetOptionsInclude(g *GetOptions, s string) *GetOptions {
+// Including ensures that the variadic refs are included in a copy of the
+// options, resulting in expansion of the the referred sub-resources. Unknown
+// values within refs will be silently ignore by the API.
+func (g *GetOptions) Including(refs ...string) *GetOptions {
```

Comment:
The Including fction is the same for all the 3 options structs. Is it too dirty to embed a "parent" stuct that would implement this `Including` only once? 

Maybe it could also just update the struct in place and not return a new value? That would make it less type-conscious and easier to generalize.

Reply:
I considered embedding the GetOptions type into ListOptions and SearchOptions when SearchOptions was introduced.  that would have affected the way people populate ListOptions in our tools (`&packngo.ListOptions{GetOptions: packngo.GetOptions{}}`) so I opted not to embed (and left comments about that in the code).

I opened #207 to merge the three types, as I no longer think there is any value in differentiating between them. Let me know your thoughts there.

As you point out, this will simplify things.

Reply:
> reaking change in the way users populate ListOptions in our tools (&packngo.ListOptions{GetOptions: packngo.GetOptions{}})

good point. I will approve this PR.